libs/template/experimental/virtual-scrolling/src/lib/util.ts

```diff
@@ -6,6 +6,10 @@ import {
 } from '@rx-angular/cdk/zone-less/browser';
 import { from, Observable } from 'rxjs';
 
+export function toBoolean(input: null | boolean | string | undefined): boolean {
+  return input != null && `${input}` !== 'false';
+}
```

Comment:
Angular exposes a [`booleanAttribute`](https://angular.io/api/core/booleanAttribute), we could use it directly.